application-confluence-migrator-pro-ui/src/main/resources/ConfluenceMigratorPro/Code/Tabs/NewMigration.xml

```diff
@@ -116,20 +116,12 @@
     #showAttachmentsLiveData($storeDoc 'confluencePackages')
 
     #if ($hasEdit || $hasAdmin)
-      ## Hide the form because it comes from the platform, and we cannot customize its appearance. We are using a 'facade' button to trigger the upload via JS.
-      &lt;form action="$storeDoc.getURL("upload")" enctype="multipart/form-data" method="post" id="addConfluencePackage"&gt;
-      &lt;div&gt;
-      ## CSRF prevention
-      &lt;input type="hidden" name="form_token" value="$!{services.csrf.getToken()}" /&gt;
-      &lt;div&gt;
-        &lt;div class="fileupload-field"&gt;
-          &lt;label class="sr-only" for="confluenceUploadFile"&gt;$escapetool.xml(
-            $services.localization.render('core.viewers.attachments.upload.file'))&lt;/label&gt;
-          &lt;input id="confluenceUploadFile" type="file" name="filepath" size="40" class="uploadFileInput noitems hidden"
-            data-max-file-size="$!escapetool.xml($xwiki.getSpacePreference('upload_maxsize'))" accept=".zip"/&gt;
-        &lt;/div&gt;
-      &lt;/div&gt;
-      &lt;/div&gt;
+      &lt;form id="addConfluencePackage"&gt;
```

Comment:
Why  did you remove `action="$storeDoc.getURL("upload")" enctype="multipart/form-data" method="post"`?